### PR TITLE
rectify function and variable referred in pkg/archive/changes_linux.go

### PR DESCRIPTION
**Pull request by lixiaobing1**
rectify function and variable referred in pkg/archive/changes_linux.go.

### CODE REVIEW

File: pkg/archive/changes_linux.go [hunk 1]
```diff
@@ -197,7 +197,7 @@ func (w *walker) walk(path string, i1, i2 os.FileInfo) (err error) {
 	return nil
 }
 
-// {name,inode} pairs used to support the early-pruning logic of the walker type
+// {name,ino} pairs used to support the early-pruning logic of the walker type
```
Comment (AkihiroSuda) [inline pkg/archive/changes_linux.go]: I'm -1 for this change, "ino" does not makes sense to me without description about "inode"

**Review by AkihiroSuda — CHANGES_REQUESTED** [review]

**Review by vdemeester — CHANGES_REQUESTED** [review]
Same as @AkihiroSuda, the only change I'm +1 on this PR is the `walkchunk`

**Review by vdemeester — APPROVED** [review]
Now LGTM 🐸
/cc @thaJeztah

**Comment by lixiaobing10051267** [comment]
yeah, that is more better, has been restored.

**Review by thaJeztah — APPROVED** [review]
LGTM

**Pull request merged**